PixelHunter1995/Player.cs

```diff
@@ -58,6 +59,24 @@ public void Update(GameTime gameTime)
         public void Draw(GraphicsDeviceManager graphics, SpriteBatch spriteBatch, Tileset tileset)
         {
             SpriteComp.Draw(graphics, spriteBatch, tileset);
+            DrawTextOverHead(spriteBatch, "Hi, I'm the player!"); // TODO: Get text from some kind of file
+        }
+
+        private Vector2 RelativePosition( int deltaX, int deltaY)
+        {
+            return new Vector2(Position.X + deltaX, Position.Y + deltaY);
+        }
+
+        private void DrawTextOverHead(SpriteBatch spriteBatch, String text)
+        {
+            int textDeltaX = -((int)Font.MeasureString(text).X / 2);
+            int textDeltaY = -((int)Font.MeasureString(text).Y + 5);
+            // Draw white around the letters to see them better
+            spriteBatch.DrawString(Font, text, RelativePosition(textDeltaX + 1, textDeltaY + 1), Color.White);
```

Comment:
If you figure out how to use a custom font that has an outline we can remove this.